Added attendee info and sales window icons (#2778)
the-events-calendar/event-tickets: readme.txt

* Tweak - Re-styled Inactive RSVP Block in Block Editor to match frontend design. Styles will be the same in block editor and in the user interface [ET-1818]
* Tweak - Re-styled Inactive Tickets Block in Block Editor. Empty state now has a new design [ET-1817]
* Tweak - Re-styled Inactive Tickets Block with tickets. Inactive state with tickets has a new design [ET-1822]

> * Tweak - Re-styled Active Tickets Block with tickets. A notes icon will appear and on hover will give information about attendee information. A sales window icon will appear and on hover will give the sales dates information  [ET-1824]

Reviewer: Might be a little too wordy.
```suggestion
* Tweak - Re-styled Active Tickets Block with tickets. Add information icons and tooltips.  [ET-1824]
```

Author: Thanks @codingmusician I have updated the readme.txt as you suggested
